pkg/trivy/report.go

```diff
@@ -99,10 +99,10 @@ type Vulnerabilities struct {
 			V2Score  float64 `json:"V2Score"`
 			V3Score  float64 `json:"V3Score"`
 		} `json:"redhat"`
-	} `json:"CVSS,omitempty"`
+	} `json:"CVSS"`
 	References       []string  `json:"References,omitempty"`
-	PublishedDate    time.Time `json:"PublishedDate,omitempty"`
-	LastModifiedDate time.Time `json:"LastModifiedDate,omitempty"`
+	PublishedDate    time.Time `json:"PublishedDate"`
```

Comment:
Maybe considering using `omitzero` instead (at least for the *date field) ?
```suggestion
	PublishedDate    time.Time `json:"PublishedDate,omitzero"`
```
from doc https://pkg.go.dev/encoding/json#Marshal :
> The "omitzero" option specifies that the field should be omitted from the encoding if the field has a zero value, according to rules:
> 1) If the field type has an "IsZero() bool" method, that will be used to determine whether the value is zero.
> 2) Otherwise, the value is zero if it is the zero value for its type.

Reply:
@Thibaut-gauvin You're right, I will add the `omitzero` option.